src/sentry/snuba/metrics/fields/base.py

```diff
@@ -149,8 +149,8 @@ def run_metrics_query(
         where=[
             Condition(Column("org_id"), Op.EQ, org_id),
             Condition(Column("project_id"), Op.IN, project_ids),
-            Condition(Column(TS_COL_QUERY), Op.GTE, start),
-            Condition(Column(TS_COL_QUERY), Op.LT, end),
+            Condition(Column(get_timestamp_column_name(entity_key.value)), Op.GTE, start),
```

Comment:
FYI, I'm going to change the gauges entity to expose `timestamp` instead of `rounded_timestamp`. I don't want to have an inconsistent API.

Reply:
Yes please, but should I wait to merge? Idk how easy it is to change that.